### PR TITLE
Add support for Android 12 splashscreen icon

### PR DESCRIPTION
**Pull request by 4ian**

### CODE REVIEW

File: newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js [hunk 1]
```diff
@@ -82,6 +85,9 @@ export default class PlatformSpecificAssetsDialog extends React.Component<
       androidIconResourceNames: androidSizes.map(size =>
         project.getPlatformSpecificAssets().get('android', `icon-${size}`)
       ),
+      androidWindowSplashScreenAnimatedIconResourceName: project
+        .getPlatformSpecificAssets()
```
Comment (ClementPasteau) [inline newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js:89]: can the `project.getPlatformSpecificAssets()` be factorised at the start of the function?

File: newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js [hunk 2]
```diff
@@ -144,6 +150,15 @@ export default class PlatformSpecificAssetsDialog extends React.Component<
             }
           )
         ),
+        resizeImage(
+          fullPath,
+          path.join(projectPath, 'android-windowSplashScreenAnimatedIcon.png'),
+          {
+            width: 576,
```
Comment (ClementPasteau) [inline newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js:157]: I assume this is the required size?

File: newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js [hunk 3]
```diff
@@ -227,6 +246,13 @@ export default class PlatformSpecificAssetsDialog extends React.Component<
         .getPlatformSpecificAssets()
         .set('android', `icon-${size}`, androidIconResourceNames[index]);
     });
+    project
+      .getPlatformSpecificAssets()
```
Comment (ClementPasteau) [inline newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js:250]: same here, do we need to call it every time?

**Review by ClementPasteau — APPROVED** [review]
lgtm! 
A screenshot to see how it looks maybe?

File: newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js (hunk 2 above)
Comment (4ian) [reply newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js]: I've added a comment and actually bumped this size so it looks good on a XXDPI screen.

File: newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js (hunk 3 above)
Comment (4ian) [reply newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js]: Fixed

File: newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js (hunk 1 above)
Comment (4ian) [reply newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js:94]: Fixed

**Comment by 4ian** [comment]
There we go:
image

**Pull request merged**